lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -0,0 +1,60 @@
+;(function (define) {
+    'use strict';
+    define(['backbone',
+            'underscore',
+            'jquery',
+            'text!templates/components/tabbed/tabbed_view.underscore',
+            'text!templates/components/tabbed/tab.underscore'],
+           function (Backbone, _, $, tabbedViewTemplate, tabTemplate) {
+               var TabbedView = Backbone.View.extend({
+                   events: {
+                       'click .nav-item': 'switchTab'
+                   },
+
+                   template: _.template(tabbedViewTemplate),
+
+                   /**
+                    * View for  a tabbed  interface. Expects a list of
+                    * tabs in its options object, each of which should
+                    * contain the following properties:
+                      * view (Backbone.View): the view to render for this tab.
```

Comment:
Nit: indentation is off. The first ``*`` should be at the same level as every other line.